lib/src/signed_video_h26x_common.c

```diff
@@ -1033,6 +1067,13 @@ hash_and_add(signed_video_t *self, const h26x_nalu_t *nalu)
     if (nalu->is_last_nalu_part) {
       // The end of the NALU has been reached. Update hash list and GOP hash.
       check_and_copy_hash_to_hash_list(self, nalu_hash, hash_size);
+      // Copy the NALU to the hash list if the NALU is not an SEI frame.
```

Comment:
Note that `hash_and_add()` is used on the signing side. What value is added to the code to store hashes in two places; `nalu_hash_list` and `hash_list`? Can't we use `hash_list` directly?

Reply:
While validating  i can put the  nalu hashes to there and compute the hash of it and compare with the |received_gop_hash|